crates/orderbook/openapi.yml

```diff
@@ -651,6 +665,31 @@ components:
       allOf:
         - $ref: "#/components/schemas/OrderCreation"
         - $ref: "#/components/schemas/OrderMetaData"
+    Auction:
+      description: |
+        A batched auction for solving.
```

Comment:
nit: I was under the impression we call them "batch auction". Could be mishearing, though.
Also wouldn't it be "A batch(ed) auction to solve".

Reply:
To cover all case we could say "An auction batch"

Reply:
I meant it to be "batch auction" - nice catch!